### ISSUE TITLE
HBbase  serialization cannot obtain the timeout period?

### ISSUE THREAD

**Issue by thomzhang** [tsuna/gohbase#214]
Why didn't HBase assign the timeout time of RequestHeader when serializing the request?
As a result, HBase serialization cannot obtain the timeout period.
https://github.com/tsuna/gohbase/blob/master/pb/RPC.proto#L130
https://github.com/tsuna/gohbase/blob/master/region/client.go#L624

**Comment by dethi** [comment]
Indeed, I have started to investigate this here https://github.com/tsuna/gohbase/pull/215

**Comment by thomzhang** [comment]
nice~

**Comment by dethi** [comment]
I haven’t got a chance yet to test if it’s working
